package.json

```diff
@@ -12,15 +12,16 @@
     "build": "gulp build",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
+    "format": "prettier --write **/*.js",
     "lint:fix": "eslint --config .eslintrc.json ./ --fix",
     "lint": "eslint --config .eslintrc.json ./",
     "precise-commits": "precise-commits",
     "precommit": "npm run precise-commits",
     "prepublishOnly": "npm run build",
     "start": "node index.js",
     "test": "npm run lint",
-    "watch": "gulp watch",
-    "format": "prettier --write **/*.js"
+    "watch": "gulp watch"
+
```

Comment:
There's an extra line here we can remove

Reply:
@DonaldLika Can you remove this extra line? Once we get this change fixed, I'll move forward and get this merged.

Thanks!